src/Core/Grid/Definition/Factory/ProductGridDefinitionFactory.php

```diff
@@ -157,7 +157,7 @@ protected function getColumns()
                         'route' => 'admin_products_v2_edit',
                         'route_param_name' => 'productId',
                         'route_param_field' => 'id_product',
-                        'fragment' => 'tab-pricing-tab',
+                        'fragment' => 'tab-product_pricing-tab',
```

Comment:
question not really related to this pr, but why is it called fragment? (this is a fragment of what?)

Reply:
It was debated for a long time actually in this PR https://github.com/PrestaShop/PrestaShop/pull/22731#discussion_r666858096 😅

But TLDR; it's the naming used by Symfony so I didn't argue back in the days:
https://github.com/symfony/symfony/blob/4.4/src/Symfony/Component/Routing/Generator/UrlGenerator.php#L297-L311

If you think it should be changed and have a better suggestion I'm open though, it's now or never 😅

Reply:
I wouldn't have question if we use the "fragment" in router (or other URL related service). But its too generic here because its Grid column, it doesn't relate directly with URL context. its like when we introduced `route_param_name` option in same Column definition - we didin't call it `param_name`. So I would suggest renaming this option to somewhat more specific, like url_fragment? route_fragment wdyt?